core/Settings/FieldConfig.php

```diff
@@ -230,4 +231,22 @@ class FieldConfig
      */
     public $condition;
 
+    /**
+     * Here you can add one or multiple instances of `Piwik\Validators\BaseValidator` to avoid having to
+     * write the same validators over and over again in {@link $validate}.
+     *
+     * Examples
+     * Want to require a value to be set?
+     * $fieldConfig->validators = new Piwik\Validators\NotEmpty();
```

Comment:
Don't want to be annoying, but you missed a validators[] here

Reply:
Will update it now... Totally forgot that one. Not annoying at all ;)